packages/gamut/src/GridForm/GridFormInputGroup/GridFormFileInput/__tests__/GridFormFileInput-test.tsx

```diff
@@ -0,0 +1,43 @@
+import GridFormFileInput from '../index';
+import { mount } from 'enzyme';
+import React from 'react';
+
+describe('GridFormFileInput', () => {
+  const type = 'file';
+  const name = 'name';
+
+  describe('when an id is passed as a prop', () => {
+    it('renders an input with the same id', () => {
+      const id = 'mycoolid';
+
+      const textInput = mount(
+        <GridFormFileInput
+          field={{
+            type: type,
+            name: name,
+          }}
```

Comment:
⛳  Nit: you could one-line these fields like:

```suggestion
          field={{ name, type }}
```

Reply:
Alternately, `packages/gamut/src/GridForm/__tests__/stubs.ts` defines a bunch of field objects you could use. Since these added tests don't rely on the contents of the fields, maybe you'd find it cleaner to use those?

Non-blocking on my end, just a mild preference.

Reply:
Thanks! Done.